### PR TITLE
Create atan2 entry for golang

### PR DESCRIPTION
**Pull request by arisdelacruz**
### Description

This is my attempt to create an entry for the atan2 function for golang. Hoping to help others on this. Thanks

### Type of Change

- Adding a new entry

### Checklist

- [ / ] All writings are my own.
- [ / ] My entry follows the Codecademy Docs style guide.
- [ / ] My changes generate no new warnings.
- [ / ] I have performed a self-review of my own writing and code.
- [ / ] I have checked my entry and corrected any misspellings.
- [ / ] I have made corresponding changes to the documentation if needed.
- [ / ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ / ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ / ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/go/concepts/math-functions/terms/atan2/atan2.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+Tags:
+  - 'Math'
+  - 'Trigonometry'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Atan2()`** function returns the arctangent value of the x/y value.
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/atan2/atan2.md:16]:
```suggestion
The **`Atan2()`** function returns the arctangent value of the *y/x* value.
```

File: content/go/concepts/math-functions/terms/atan2/atan2.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+## Syntax
+
+```pseudo
+result := math.Atan2(y,x)
+```
+
+Notes on `Atan2()`:
+
+- The ratio of the two arguments `(y/x)` is used to calculate the value of the arc tangent.
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/atan2/atan2.md:26]:
```suggestion
- The ratio of the two arguments `(y/x)` is used to calculate the value of the arctangent.
```

File: content/go/concepts/math-functions/terms/atan2/atan2.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+- The `result` will be between between -π/2 to π/2 radians.
+- The value returned will be of type `float64`.
+
+## Special Cases
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/atan2/atan2.md:30]:
```suggestion
### Special Cases
```

File: content/go/concepts/math-functions/terms/atan2/atan2.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+`NAN:`
+
+- Atan2(y, NaN) = NaN
+- Atan2(NaN, x) = NaN
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/atan2/atan2.md:35]:
```suggestion
`NAN`:
- *Atan2(y, NaN) = NaN*
- *Atan2(NaN, x) = NaN*
```

Comment (caupolicandiaz) [reply content/go/concepts/math-functions/terms/atan2/atan2.md:35]: Update the others as above

File: content/go/concepts/math-functions/terms/atan2/atan2.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+## Example
+
+The following code calculates the arctangent and prints out the result:
+
+```go
+package main
+
+import "fmt"
+import  "math"
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/atan2/atan2.md:75]:
```suggestion
import (
  "fmt"
  "math"
  )
```

Comment (caupolicandiaz) [reply content/go/concepts/math-functions/terms/atan2/atan2.md:75]: make sure indentations in the code use 2 spaces not tabs

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@arisdelacruz I've added my initial comments. Please update the entry, and tag me when you have finished making changes. Thanks

**Comment by arisdelacruz** [comment]
@caupolicandiaz 

done, buddy

thanks

**Review by caupolicandiaz — APPROVED** [review]
@arisdelacruz This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@arisdelacruz, @caupolicandiaz Looks good to me.

**Pull request merged**